src/navigation/components/Header/Header.tsx

```diff
@@ -68,9 +70,10 @@ const isDropdownItem = (item: NavigationItemModel): item is NavigationDropdownIt
     item.type === NavigationItemType.Dropdown;
 
 export const Header: React.FC<HeaderProps> = ({data, logo}) => {
-    const {leftItems, rightItems, iconSize = 20} = data;
+    const {leftItems, rightItems, iconSize = 20, withBorder = false} = data;
     const [isSidebarOpened, setIsSidebarOpened] = useState(false);
     const [activeItemId, setactiveItemId] = useState<string | undefined>(undefined);
+    const [withHeaderBorder, setWithHeaderBorder] = useState(withBorder);
```

Comment:
what is the difference between withBorder and withBorder?

Reply:
- We pass the prop `withBorder` which can show or hide border when `scrollY = 0`. 
- We always have to show border when `scrollY > 0`. 
- I chose not to rewrite the original prop to hide/show border but create a new one `withHeaderBorder` and change this state if it's necessary. I rename variable `withHeaderBorder` to `showBorder` is that better?